packages/vite/src/node/__tests__/utils.spec.ts

```diff
@@ -243,8 +243,10 @@ describe('isFileReadable', () => {
       expect(isFileReadable(testFile)).toBe(true)
     })
     test('file without read permission', async () => {
-      fs.chmodSync(testFile, '044')
-      expect(isFileReadable(testFile)).toBe(false)
+      if (process.getuid && process.getuid() !== 0) {
+        fs.chmodSync(testFile, '044')
+        expect(isFileReadable(testFile)).toBe(false)
+      }
       fs.chmodSync(testFile, '644')
```

Comment:
Should we skip the test entirely instead? Otherwise it's only running `chmodSync` if the if condition fails. Something like `test.runIf(process.getuid && process.getuid() !== 0)('...'`

Reply:
Thanks for your suggestion, I also don't know whether we need to skip the whole test, I am not sure about the significance of this test for the whole vite, so I leave the decision of whether this code needs to be changed to the vite maintainer

Reply:
There's two other maintainers who put a 👍 to my comment so it's likely that we want this change.

Reply:
> There's two other maintainers who put a 👍 to my comment so it's likely that we want this change.

Thank you for your help. I changed `if` to `test.runIf` in the new commit. Please let me know if there is any deficiency, I will be happy to modify this part of the code again